.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
Think a StackOverflow one might be good in here too. I'm not sure if there is an existing tag for that though

Reply:
Addressed in the next commit.